fstest/test_all/config.yaml

```diff
@@ -388,3 +388,14 @@ backends:
    fastlist: false
    ignore:
      - TestRWFileHandleWriteNoWrite
+ - backend: "oracleobjectstorage"
+   remote: "TestOracleObjectStorage:"
+   fastlist: true
+   ignore:
+     - TestIntegration/FsMkdir/FsEncoding/control_chars
+     - TestIntegration/FsMkdir/FsPutFiles/SetTier
```

Comment:
See above for setting the correct Tiers in the tests. If the backend doesn't support tiers then support for them should be removed.

Reply:
done added support for tiers